src/core/connection.h

```diff
@@ -194,6 +194,11 @@ typedef union QUIC_CONNECTION_STATE {
         //
         BOOLEAN DisableVneTp : 1;
 #endif
+
+        //
+        // Indicates whether this connection offloads encryption workload to HW
+        //
+        BOOLEAN EncryptionOffloading : 1;
```

Comment:
I _think_ we might need to set this per-path, not connection-wide (or at least not _just_ connection-wide) because when we set for each path, it could fail on one path and not the other depending on HW resources.